frontends/web/src/routes/account/add/addaccount.tsx

```diff
@@ -0,0 +1,90 @@
+/**
+ * Copyright 2018 Shift Devices AG
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import { h, RenderableProps } from 'preact';
+import { Button, Input, Select } from '../../../components/forms';
+import { Entry } from '../../../components/guide/entry';
+import { Guide } from '../../../components/guide/guide';
+import Header from '../../../components/header/Header';
+import { translate, TranslateProps } from '../../../decorators/translate';
+import { apiPost } from '../../../utils/request';
+
+function submit() {
+    const body = {
+        coin: (document.getElementById('coin') as HTMLSelectElement).value,
```

Comment:
I would either:

- use [controlled form elements](https://reactjs.org/docs/forms.html) for consistency (it is done that way with every other form in our frontend, I believe)
- or use refs to access the value instead of `id`s (IDs can collide and are hard to track, more easy to break silently, etc.) like described here: https://reactjs.org/docs/uncontrolled-components.html

Reply:
> use controlled form elements for consistency (it is done that way with every other form in our frontend, I believe)

👍

Reply:
Fully agreed. I just hacked the Xpub-Import-MVP together yesterday.

I already collected a lot of outstanding tasks in https://github.com/shiftdevices/godbb/issues/618 and created issues for each of them now.

I wanted to leave form improvements to our frontend experts, which was for me part of `improve the add account route`. I made this issue now explicit (https://github.com/shiftdevices/godbb/issues/634) and will do it myself because I wanted to try https://preactjs.com/guide/linked-state for quite some time. 🤓

Conclusion: I merge this PR now and leave all further improvements, with which I fully agree, to future PRs.